app/configurations/config.okc.js

```diff
@@ -52,7 +52,7 @@ export default configMerger(walttiConfig, {
     PELIAS_REVERSE_GEOCODER: `${GEOCODING_BASE_URL}/reverse`,
     PELIAS_PLACE: `${GEOCODING_BASE_URL}/place`,
     FONT: null,
-    
+    ROUTE_PDF: ROOTLINK+'/assets/documents/Maps-Schedules/Route-{route.shortName}-Schedule-Map.pdf',
```

Comment:
I think storing non-`digitransit-ui` assets under the same domain/host is a bad idea. In there future, with upstream merges, new URL conflicts may appear, and we won't notice them properly.

I strongly suggest to move either `digitransit-ui` or the "general" assets to a different host (e.g. `static.go.embarkok.com` or `static.embarkok.com`).

Reply:
We'll tackle this soon, but not as part of this PR.